front/src/views/Boxes/components/BoxesTable.tsx

```diff
@@ -165,6 +167,33 @@ function BoxesTable({
       });
     }
 
+    // Sync filters with URL parameters
+    filters.forEach((filter) => {
+      switch (filter.id) {
+        case "location":
+          updateFilter("location_id", Array.isArray(filter.value) ? filter.value[0] : filter.value);
+          break;
+        case "productCategory":
+          updateFilter("category_id", Array.isArray(filter.value) ? filter.value[0] : filter.value);
+          break;
+        case "product":
+          updateFilter("product_id", Array.isArray(filter.value) ? filter.value[0] : filter.value);
+          break;
+        case "size":
+          updateFilter("size_id", Array.isArray(filter.value) ? filter.value[0] : filter.value);
+          break;
+        case "gender":
+          updateFilter("gender_id", Array.isArray(filter.value) ? filter.value[0] : filter.value);
+          break;
+        case "state":
+          updateFilter("box_state", Array.isArray(filter.value) ? filter.value : [filter.value]);
+          break;
+        case "tags":
+          updateFilter("tag_ids", Array.isArray(filter.value) ? filter.value : [filter.value]);
```

Comment:
This doesn't work yet. When I apply a tag filter in the table, I see `http://localhost:3000/bases/2/boxes?box_state=InStock&tag_ids=%5Bobject+Object%5D` as URL, meaning the tags object is not propertly converted to IDs